pkg/reconciler/intevents/reconciler.go

```diff
@@ -288,5 +288,6 @@ func (psb *PubSubBase) DeletePubSub(ctx context.Context, pubsubable duck.PubSuba
 		return fmt.Errorf("failed to delete PullSubscription: %w", err)
 	}
 	status.SinkURI = nil
+	status.SubscriptionID = ""
```

Comment:
Any reason not to move it to after line 281? These are all pubsub related fields IIUC.

Reply:
the subscriptionID was originally set up by the pullSubscription (only after pullSubscription turns ready), https://github.com/google/knative-gcp/blob/master/pkg/reconciler/intevents/reconciler.go#L198,
so it should be unset if pullsubscription is fully deleted.

Line 238 is for topic.